### PR TITLE
Feature/3704/remove restub button in checker workflow

### PR DESCRIPTION
**Pull request by wendidy**
https://github.com/dockstore/dockstore/issues/3704
Inside my workflows, there used to be a "restub" button on workflows and checker workflows. But now we would like to remove that button on checker workflows, this PR addresses this.

(Currently the restub button is broken even outside of checker workflow, already created a ticket on this: https://github.com/dockstore/dockstore/issues/4193)

### CODE REVIEW

**Review by garyluu — COMMENTED** [review]
Could use an explanation on what was changed (why it's more than a 10 character PR)

**Review by coverbeck — APPROVED** [review]

File: src/app/workflow/info-tab/info-tab.component.html [hunk 1]
```diff
@@ -204,22 +212,16 @@
...
           <!-- TODO: Remove once hosted workflows and Nextflow support checker workflows -->
           <div *ngIf="workflow?.mode === WorkflowType.ModeEnum.HOSTED || (isNFL$ | async)">
             <strong matTooltip="Currently disabled for Hosted Workflows and Nextflow Workflows">Checker Workflow</strong>: n/a
           </div>
-          <li *ngIf="!isPublic">
-            <strong [matTooltip]="modeTooltipContent">Mode</strong>: {{ workflow?.mode }}
-            <button
-              class="btn btn-link push-right"
-              type="button"
-              *ngIf="workflow?.mode === WorkflowType.ModeEnum.FULL && !workflow?.is_published"
```
Comment (garyluu) [inline src/app/workflow/info-tab/info-tab.component.html:217]: Why is this PR not just a simple `*ngIf="workflow?.mode === WorkflowType.ModeEnum.FULL && !workflow?.is_published && !workflow?`.isChecker"

**Review by garyluu — CHANGES_REQUESTED** [review]
Ctrl-f `Mode<` returns 4 results. Previously it was pretty clear that it's always shown on non-public page, now the occurence has been tripled and unclear.

Info-tab-checker-workflow-path.component is only meant to have the single checker row in it. Having mode sometimes appear inside it and sometimes not is also confusing.

File: src/app/workflow/info-tab/info-tab.component.html (hunk 1 above)
Comment (wendidy) [reply src/app/workflow/info-tab/info-tab.component.html:217]: 😨 ....bruh, I wasn't aware of this...

**Comment by wendidy** [comment]
fix is easy like Gary said, will create a new branch of this, closing

**Pull request closed without merging**